pdns/dnsdistdist/docs/guides/webserver.rst

```diff
@@ -9,6 +9,13 @@ To visually interact with dnsdist, try add :func:`webserver` to the configuratio
 
 Now point your browser at http://127.0.0.1:8083 and log in with any username, and that password. Enjoy!
 
+Since 1.5.0, only connections from 127.0.0.1 and ::1 are allowed by default. To allow connections from 192.0.2.1, instead:
```

Comment:
This is actually a list of netmasks. So this would probably be better to document here and after that it accepts various forms (eg. `192.0.2.1`, `192.0.0.0/16`, and even `!192.0.0.0/16`)